### DISCUSSION TITLE
Ability to see your highest WPM for the specific test

### DISCUSSION THREAD

**Discussion by ObiKeahloa** [monkeytypegame/monkeytype#1027] [Ideas] [answered]
Is it possible to add an ability to be able to see your highest WPM during that test beside your raw speed (The Highest WPM after the first few words as when starting you type very fast then slowly slow down similar to how it will show our Personal Best in the graph is it possible to show out highest ?)

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
I'm kind of confused as to what your asking here. You kind of mentioned this but you can easily see your highest WPM for the test in the graph so I don't see the point in adding another variable to keep track of.

**Reply by ObiKeahloa** [reply]
I am saying that it would be cool if we could see it without having to refer the graph.

**Reply by frogfrogfrogfrogfrog** [reply]
Yeah, but what would be the point of adding that number? Everything on the result screen has a purpose that another thing on there doesn't have. So if @Miodec added a number that said your highest wpm we would have a number that does something the graph already does.

**Reply by Miodec** [reply]
Agree with @TheeKevin - just check the graph. There is already a lot of numbers on the result screen.

**Comment by jweickm** [comment]
While this is an old discussion that has already been closed, I still want to post my question here. I have not been able to find my personal best WPM on the results screen since a few weeks. It used to be there and despite not changing any settings pertaining to that, I do not see the personal best WPM info anymore. I have also checked the settings, just in case. Has this feature been disabled or is there another solution I'm not aware of?
Help is appreciated,
jweickm
